layouts/partials/navbar-version-selector.html

```diff
@@ -3,8 +3,8 @@
 </a>
 <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownMenuLink">
 	{{ $p := . }}
-	<a class="dropdown-item" href="/releases">Release Information</a>
+	<a class="dropdown-item" href="{{ "releases" | relLangURL}}">{{ i18n "release_information_navbar" . }}</a>
```

Comment:
```suggestion
	<a class="dropdown-item" href="{{ "releases" | relLangURL }}">{{ i18n "release_information_navbar" . }}</a>
```
Sorry I could not help myself to propose to add that useless space 😛

Reply:
😆 Thanks! should be updated now